recipe-box-frontend/src/main/frontend/src/Pages/LoginPage/UserLogin.css

```diff
@@ -0,0 +1,110 @@
+.login-card {
+    width: 460px;
+    height: 715px;
+    padding: 50px;
+    background-color: white;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    display: flex;
+    flex-direction: column;
+
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+.close-button {
+    position: absolute;
+    top: 20px;
+    right: 20px;
+    background-color: transparent;
+    border-radius: 5px;
+    padding: 10px;
+    transition: background-color 0.3s ease;
+}
+.close-button:hover {
+    background-color: lightgray;
+}
+
+.login-logo {
+    margin: 100px auto 65px;
+    width: 350px;
+    height: auto;
+}
+.input-group {
+    width: 100%;
+    margin-bottom: 30px;
+    display: flex;
+    flex-direction: column;
+}
+
+.input-group.error input {
+    border-color: red;
+}
+
+.input-group.error .error-text {
+    color: red;
+}
+
+.input-group div {
+    margin-bottom: 15px;
+}
+
+.input-group input {
+    padding: 0px;
+    height: 45px;
+    border: 1px solid lightgray;
+    border-radius: 5px;
+    font-weight: normal;
+}
+
+.input-group input[type="text"] {
+    background: white url(../../assets/icons/person_icon.png) left no-repeat;
+    background-size: 2rem 1.5rem;
+    padding-left: 50px;
+    background-position: 3px center;
+}
+.input-group input[type="password"] {
+    font-size: 2.5rem;
```

Comment:
this font size is bigger than the one for the username, try making it consistent